cmd/nodepool/kubevirt/create.go

```diff
@@ -61,7 +80,7 @@ func (o *KubevirtPlatformCreateOptions) UpdateNodePool(_ context.Context, nodePo
 							Devices: kubevirtv1.Devices{
 								Disks: []kubevirtv1.Disk{
 									{
-										Name: "containervolume",
+										Name: util.RHCOSMagicVolumeName,
 										DiskDevice: kubevirtv1.DiskDevice{
 											Disk: &kubevirtv1.DiskTarget{
 												Bus: "virtio",
```

Comment:
I think we need to do the same sort of disk/volume defaulting here as in the api/fixtures/example.go ? It's unclear to me why we have two locations where it looks like the VirtualMachineInstance is specified for the create command.